static/script.js

```diff
@@ -160,24 +199,37 @@ app.controller('chiveCtrl', function($scope, $http, $location, $rootScope) {
 
 		
 		// end bug
-		
 
-		
+
+
+
 		$.ajax({
-		    url: 'http://localhost:5000/user?first_name=' + firstName + '&last_name=' + lastName + '&username=' + username + '&email=' + email + '&schoolName=' + school + '&schoolAddress=' + schoolAddress,
+		    url: 'http://localhost:5000/user',
```

Comment:
Nice! 👍

Reply:
Does the python route need to change too?

Reply:
I think it would work if I kept the same URL because we are still accessing the `/user` route, but it seems to work in Python because there are different methods defined for the `/user` route for different HTTP methods. 

I might be misunderstanding your question, though.

Reply:
Ah - looks like i just missed the change. this looks right:
```
@app.route('/user', methods=['POST'])
def create_user():
    if request.mimetype != 'application/json':
        raise Exception('Content-Type is not "application/json".')
    j = request.get_json()
    Session, engine = dbconnect(db_options)
    session = Session()
    user = User(
        first_name=j.get('first_name'),
        last_name=j.get('last_name'),
        username=j.get('username'),
        password=hashString(j.get('password')),
        school_id=get_or_create_school(session, j.get('school')).id,
        email=j.get('email')
    )
```